Gemfile

```diff
@@ -60,7 +60,7 @@ group :development, :test do
   gem 'rspec-rails', '~> 3.3'
   gem 'thin'
   gem 'bullet'
-  gem 'mailcatcher', '0.6.3'
```

Comment:
Hmm... I think I had issues with the latest mail catcher.. but I can't remember exactly what they were.